packages/blade/src/components/Charts/DonutChart/DonutChart.web.tsx

```diff
@@ -104,16 +129,29 @@ const ChartDonutWrapper: React.FC<ChartDonutWrapperProps & TestID & DataAnalytic
               fontWeight={theme.typography.fonts.weight.bold}
               letterSpacing={theme.typography.letterSpacings[100]}
               dy={
-                content?.label
+                isLabelPresentInContent
                   ? LABEL_DISTANCE_FROM_CENTER[pieChartRadius].withLabel
                   : LABEL_DISTANCE_FROM_CENTER[pieChartRadius].normal
               }
             >
-              {content?.text}
+              {content?.value}
             </Label>
           )}
         </RechartsPieChart>
       </RechartsResponsiveContainer>
+
+      {isValidElement(content) && (
+        <BaseBox
+          position="absolute"
+          top="50%"
+          left="50%"
+          transform={`translate(-50%, calc(-50% - ${legendHeight / 2}px))`}
+          zIndex={10}
+          textAlign="center"
+        >
+          {content}
+        </BaseBox>
```

Comment:
The legends are always at the bottom right? any scenario in Donut chart where legends might be on right side or left side? will this calculation break there?

<img width="543" height="332" alt="image" src="https://github.com/user-attachments/assets/38ea57c0-b4bf-4c62-8b65-b9e253c938e7" />